modern/src/LoginPage.js

```diff
@@ -114,27 +127,42 @@ const LoginPage = () => {
             value={password}
             type='password'
             autoComplete='current-password'
-            onChange={handlePasswordChange} />
-
-          <FormControl fullWidth margin='normal'>
-            <div className={classes.buttons}>
-              <Button type='button' variant='contained' onClick={handleRegister} disabled={!registrationEnabled}>
+            onChange={handlePasswordChange}
+            variant='filled' />
+          <Button 
+            type='submit'
+            variant='contained' 
+            color='secondary'
+            className={classes.submit}
+            disabled={!email || !password}
+            fullWidth>
+            {t('loginLogin')}
+          </Button>
+          <Grid container>
+            <Grid item xs={3}>
```

Comment:
1. Kinldy remove `xs={3}` so the width of the grid will adjust to the content.
1. You'll need to change the grid for the language button (line 147) to `<Grid item xs>` in order for it to auto-fill the rest of the space

Reply:
Done.